yt_dlp/YoutubeDL.py

```diff
@@ -4152,10 +4152,11 @@ def urlopen(self, req):
                 if (
                     'unsupported proxy type: "https"' in ue.msg.lower()
                     and 'requests' not in self._request_director.handlers
+                    and 'niquests' not in self._request_director.handlers
```

Comment:
```suggestion
                    and 'Niquests' not in self._request_director.handlers
```

(this is a bug I'm fixing in another PR, but just to prepare for the patch...)

Reply:
Noted.